testing_and_setup/compass/manage_regression_suite.py

```diff
@@ -123,13 +126,13 @@ def process_test_setup(test_tag, config_file, work_dir, model_runtime,
                 command)
 
             # Write test case run step
-            suite_script.write("print ' ** Running case {}'\n".format(
+            suite_script.write("print(' ** Running case {}'\n".format(
                 test_name))
             suite_script.write('try:\n')
             suite_script.write('    {}\n'.format(command))
-            suite_script.write("    print '      PASS'\n")
+            suite_script.write("    print('      PASS'\n")
```

Comment:
Missing right parenthesis. Same for all print statements in this file where only left parenthesis is added.

Reply:
fixed.